CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.4 (07 October 2016)
+
+* Add ability to remove repository location (#86)
+* Fixed bug in connection parsing when federated connections are present (#87)
```

Comment:
oops, too quick to hit the merge button.  I'll send a second one that includes this.  For some reason I thought #80 shipped in 0.3